stories/card/card.stories.js

```diff
@@ -221,7 +221,7 @@ export const analyticalCard = () => `
                             </div>
                             <div class="fd-numeric-content__scale-container">
                                 <div class="fd-numeric-content__scale">
-                                    <span class="fd-numeric-content__scale-arrow sap-icon--down"></span>
+                                    <i class="fd-numeric-content__scale-arrow sap-icon--down" aria-label="decrease"></i>
```

Comment:
Shouldn't we have `role="presentation"`?

Reply:
In this case `aria-label` is needed, because the icon provides some information important for the user.